### ISSUE TITLE
The values must be checked, so NULL and undefined will not be accepted

### ISSUE THREAD

**Issue by obodeldog** [ioBroker/ioBroker.sql#84]
```
2019-07-27 21:52:57.361  - [31merror[39m: sql.0 uncaught exception: Cannot read property 'toString' of undefined
2019-07-27 21:52:57.361  - [31merror[39m: sql.0 TypeError: Cannot read property 'toString' of undefined
    at Object.exports.insert (/opt/iobroker/node_modules/iobroker.sql/lib/mysql.js:29:66)
    at pushValueIntoDB (/opt/iobroker/node_modules/iobroker.sql/main.js:1498:26)
    at Timeout.pushHelper [as _onTimeout] (/opt/iobroker/node_modules/iobroker.sql/main.js:1175:9)
    at ontimeout (timers.js:505:15)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
2019-07-27 21:52:57.363  - [33mwarn[39m: sql.0 Exception: TypeError: Cannot read property 'toString' of undefined
2019-07-27 21:52:57.391  - [31merror[39m: sql.0 uncaught exception: Cannot read property 'toString' of undefined
2019-07-27 21:52:57.391  - [31merror[39m: sql.0 TypeError: Cannot read property 'toString' of undefined
    at Object.exports.insert (/opt/iobroker/node_modules/iobroker.sql/lib/mysql.js:29:66)
    at pushValueIntoDB (/opt/iobroker/node_modules/iobroker.sql/main.js:1498:26)
    at Timeout.pushHelper [as _onTimeout] (/opt/iobroker/node_modules/iobroker.sql/main.js:1175:9)
    at ontimeout (timers.js:505:15)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
2019-07-27 21:52:57.391  - [33mwarn[39m: sql.0 Exception: TypeError: Cannot read property 'toString' of undefined
2019-07-27 21:52:57.392  - [31merror[39m: sql.0 uncaught exception: Cannot read property 'toString' of undefined
2019-07-27 21:52:57.392  - [31merror[39m: sql.0 TypeError: Cannot read property 'toString' of undefined
    at Object.exports.insert (/opt/iobroker/node_modules/iobroker.sql/lib/mysql.js:29:66)
    at pushValueIntoDB (/opt/iobroker/node_modules/iobroker.sql/main.js:1498:26)
    at Timeout.pushHelper [as _onTimeout] (/opt/iobroker/node_modules/iobroker.sql/main.js:1175:9)
    at ontimeout (timers.js:505:15)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
    at listOnTimeoutNT (timers.js:369:26)
    at _combinedTickCallback (internal/process/next_tick.js:136:11)
    at process._tickDomainCallback (internal/process/next_tick.js:219:9)
2019-07-27 21:52:57.392  - [33mwarn[39m: sql.0 Exception: TypeError: Cannot read property 'toString' of undefined
2019-07-27 21:52:57.865  - [32minfo[39m: sql.0 terminating
```

or

```sql.0 Cannot insert INSERT INTO `iobroker`.ts_number (id, ts, val, ack, _from, q) VALUES(391, 1564257209015, undefined, 1, 1, 0);: Error: ER_BAD_FIELD_ERROR: Unknown column 'undefined' in 'field list'```

**Comment by Apollon77** [comment]
Which version do you use? GitHub or an official npm version?

**Comment by Apollon77** [comment]
will be fixed in 0.11
